demo-shell/src/app/components/task-list-demo/task-list-demo.component.html

```diff
@@ -128,6 +142,8 @@
         [start]="start"
         [page]="page"
         [size]="size"
+        [dueDate]="dueDate"
```

Comment:
why we are calling the fields  
[dueDate]="dueDate"
        [endDate]="endDate" 
if on the service side they are called
dueAfter: string;
    dueBefore: string;